fix:[CDS-95390]: Add info regarding export KUBECONFIG (#6482)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/cd-steps/utilities/shell-script-step.md

kubectl scale deploy -n <+infra.namespace> $(kubectl get deploy -n <+infra.namespace> -o jsonpath='{.items[?(@.spec.selector.matchLabels.harness\.io/color=="'$(kubectl get service/<+pipeline.stages.nginx.spec.execution.steps.stageDeployment.output.stageServiceName> -n <+infra.namespace> -o jsonpath='{.spec.selector.harness\.io/color}')'")].metadata.name}') --replicas=0
```

:::info note

> The export ``KUBECONFIG` is only needed if the delegate is not installed in the target cluster. If your delegate is already installed in the target cluster, you can skip adding this export to your shell script.

Reviewer: ```suggestion
You need not export ``KUBECONFIG` to your shell script if you've already installed delegate on the target cluster.
```
